.github/workflows/test-sdk-packages.yml

```diff
@@ -0,0 +1,24 @@
+name: Test Packaged SDKs
+
+on:
+  pull_request:
+    paths:
+      - ufc/**
+  push:
+    branches:
+      - main
+      - tp/sdktest/**
+  workflow_dispatch:
+
+jobs:
+  test-php-sdk:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: ['linux']
+    uses: ./.github/workflows/test-server-sdk.yml
+    with:
+      platform: ${{ matrix.platform }}
+      sdkName: 'eppo/php-sdk'
```

Comment:
very nice - my understanding of how this fits together is the `test-server-sdk.yml` is templatized and we will add additional SDKs here as they become available

Reply:
@leoromanovsky you got it!
I tried to configure it with the SDK as a dimension in the runner matrix, but each language needs to test against different dimensions and the job matrix doesn't work like that.